### PR TITLE
Support for Blink camera system

### PR DESCRIPTION
**Pull request by fronzbot**
**Description:**
Documentation for the blink camera system.  Wasn't sure if adding the switch/camera/sensor components were necessary since they just point back to the main blink component, but I added them anyways since it seems a lot of existing components follow that structure.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#6444

### CODE REVIEW

File: source/_components/blink.markdown [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+sharing: true
+footer: true
+logo: blink.png
+ha_category: Hub
+ha_release: "0.40"
+---
+
+The `blink` component lets you view camera images and motion events from [Blink](http://blinkforhome.com) camera and security systems.
+
+You will need your blink login information (username, usually you email address, and password) to use this module.
```
Comment (Landrash) [inline source/_components/blink.markdown]: Capital letter on Blink.

File: source/_components/blink.markdown [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+```yaml
+# Example configuration.yaml entry
+blink:
+  username: YOUR_USERNAME
+  password: YOUR_PASSWORD
+```
+
+Once loaded, your front end will have the following components:
+* A camera image for each camera in your system
```
Comment (Landrash) [inline source/_components/blink.markdown]: . on the end i missing for each item in this list.

**Comment by Landrash** [comment]
@fronzbot Sorry about that. There's no real way for me to followup on what been merged and not. Hopefully the merger in the main repo chooses to merge the docs most of the time that why I mark them as I do :)

**Comment by balloob** [comment]
Sorry, I should get better at that.

**Comment by robmarkcole** [comment]
Is it possible to show an example of how to integrate this component into an automation? Currently struggling with this on https://community.home-assistant.io/t/help-with-switch-to-toggle-alarm/15276/5

**Pull request merged**